check_nextcloud.php

```diff
@@ -6,12 +6,13 @@
  * Monitoring plugin to check the status of nextcloud serverinfo app
  *
  * Copyright (c) 2019 Kevin Köllmann <mail@kevinkoellmann.de>
- *
+ * contribution: Conrad Beckert <kontakt@miradata.de>
```

Comment:
Your contribution will already be mentioned in the README and on the GitHub project itself, so it's not needed here in the actual script, too.